simpleMail.py

```diff
@@ -49,7 +49,7 @@ def send_email(frm, to, subject, message, attachments=None, maxsize=100000, html
     msg['From'] = m_from
     # if we're passed a string on 'to', don't join with commas.  If we're passed a list,
     # we'll need to join the adddresses with commas.
-    if not hasattr(m_to, 'upper'):
+    if isinstance(m_to, list):
```

Comment:
Nice.  This is cleaner.  I've always used hasattr(<obj>, 'upper'), because other instances of items could be iterable, besides lists. :+1: